src/main/kotlin/app/revanced/patches/youtube/layout/hide/shorts/resource/patch/HideShortsComponentsResourcePatch.kt

```diff
@@ -0,0 +1,109 @@
+                    SwitchPreference(
+                        "revanced_hide_shorts_join_button",
+                        StringResource("revanced_hide_shorts_join_button_title", "Hide join button"),
+                        false,
+                        StringResource("revanced_hide_shorts_join_button_on", "Join button is hidden"),
+                        StringResource("revanced_hide_shorts_join_button_off", "Join button is shown")
+                    ),
+                    SwitchPreference(
+                        "revanced_hide_shorts_subscribe_button",
+                        StringResource("revanced_hide_shorts_subscribe_button_title", "Hide subscribe button"),
+                        false,
+                        StringResource("revanced_hide_shorts_subscribe_button_on", "Subscribe button is hidden"),
+                        StringResource("revanced_hide_shorts_subscribe_button_off", "Subscribe button is shown")
+                    ),
+                    SwitchPreference(
+                        "revanced_hide_shorts_thanks_button",
+                        StringResource("revanced_hide_shorts_thanks_button_title", "Hide thanks button"),
+                        false,
+                        StringResource("revanced_hide_shorts_thanks_button_on", "Thanks button is hidden"),
+                        StringResource("revanced_hide_shorts_thanks_button_off", "Thanks button is shown")
+                    ),
+                    SwitchPreference(
+                        "revanced_hide_shorts_comments_button",
+                        StringResource("revanced_hide_shorts_comments_button_title", "Hide comments button"),
+                        false,
+                        StringResource("revanced_hide_shorts_comments_button_on", "Comments button is hidden"),
+                        StringResource("revanced_hide_shorts_comments_button_off", "Comments button is shown")
+                    ),
+                    SwitchPreference(
+                        "revanced_hide_shorts_remix_button",
+                        StringResource("revanced_hide_shorts_remix_button_title", "Hide remix button"),
+                        false,
+                        StringResource("revanced_hide_shorts_remix_button_on", "Remix button is hidden"),
+                        StringResource("revanced_hide_shorts_remix_button_off", "Remix button is shown")
+                    ),
+                    SwitchPreference(
+                        "revanced_hide_shorts_share_button",
+                        StringResource("revanced_hide_shorts_share_button_title", "Hide share button"),
+                        false,
+                        StringResource("revanced_hide_shorts_share_button_on", "Share button is hidden"),
+                        StringResource("revanced_hide_shorts_share_button_off", "Share button is shown")
+                    ),
+                    SwitchPreference(
+                        "revanced_hide_shorts_like_button",
```

Comment:
Hiding `like` & `dislike` button aren't implemented in [integrations](https://github.com/revanced/revanced-integrations/pull/395)!